core/src/main/mima-filters/1.1.x.backwards.excludes/slick.backwards.excludes

```diff
@@ -1,7 +1,42 @@
+ProblemFilters.exclude[IncompatibleSignatureProblem]("org.apache.pekko.persistence.jdbc.db.EagerSlickDatabase.unapply")
```

Comment:
can you add a license header like https://github.com/apache/incubator-pekko-connectors/pull/223/files#diff-f40f47eb001aeab00857490311b5a573779b3088aaa6ebc0b7a3dc80fd900d89 ?